index.html

```diff
@@ -29,84 +29,68 @@
-          <h3>Maximize simplicity</h3>
-          <p><em>If you don’t actively fight for simplicity in software, complexity will win, and it will suck.</em></p>
-          <ul>
-            <li>Everything is a CommonJS module.</li>
-            <li>Everything is installed via npm.</li>
-            <li>Everything is MIT licensed.</li>
-            <li>Trusted versioning with semver.</li>
-          </ul>
-          <h3>Designed for rich “app” experiences</h3>
-          <ul>
-            <li>Modern browsers by default. Unapologetically supporting only IE9+.</li>
-            <li>The browser is a runtime, not a document viewer.</li>
-            <li><3s realtime apps</li>
-          </ul><a href="/learn" class="button button-primary">Discover Ampersand.js</a>
-        </div>
-        <div class="releases-list">
-          <h3>Recent releases</h3>
-          <div class="module-card">
-            <h4 class="module-name">ampersand</h4>
-            <p class="module-version">v3.0.1 – Apr 21, 2015</p>
-            <p class="module-links"><a href="https://github.com/ampersandjs/ampersand">github</a><a href="https://www.npmjs.org/package/ampersand">npm</a></p>
-          </div>
-          <div class="module-card">
-            <h4 class="module-name">ampersand-router</h4>
-            <p class="module-version">v3.0.2 – Apr 08, 2015</p>
-            <p class="module-links"><a href="https://github.com/ampersandjs/ampersand-router">github</a><a href="https://www.npmjs.org/package/ampersand-router">npm</a></p>
-          </div>
-          <div class="module-card">
-            <h4 class="module-name">ampersand-select-view</h4>
-            <p class="module-version">v3.0.1 – Apr 08, 2015</p>
-            <p class="module-links"><a href="https://github.com/AmpersandJS/ampersand-select-view">github</a><a href="https://www.npmjs.org/package/ampersand-select-view">npm</a></p>
-          </div><a href="/contribute" class="button button-primary">How to contribute</a>
-        </div>
+        <h2>What is Ampersand.js?</h2>
+        <p>Imagine a version of <a href="http://backbonejs.org/">Backbone.js</a> where each piece (model, view, collection, etc) is its own <a href="http://github.com/ampersandjs">GitHub repository</a> and <a href="https://www.npmjs.com/search?q=ampersand">npm module</a> with good, unified <a href="/docs">documentation</a> (and yes, npm is <em>wonderful</em> for frontend code).</p>
+        <p><em>But</em>, it's not just that. We added features to some of those pieces and <a href="http://ampersandjs.com/docs#ampersand-state">replaced others entirely</a> but if you know Backbone.js you'll feel at home.</p>
+        <p>Companies like <a href="https://www.whatsapp.com/">WhatsApp</a>, <a href="http://www.flipkart.com/">FlipKart</a>, and <a href="#who">others</a> have shipped apps at incredible scale using various Ampersand modules.</p>
+        <p>There is no "core" module. You <code>npm install</code> only what you need.</p>
```

Comment:
I'd say this section could live on this own. Saying something like "How to install" and then having this below, or slightly rephrased. That's what I'd expect. Otherwise  you have to skim a bunch of content to find instructions on how to use.